WikiWatershed/model-my-watershed
# Show summary of modification on hover

To test:
 * In Modeling view, in Land Uses drop-down, drag the mouse over different Land Use icons. 
 * The titles and summaries should update properly, and clear when the cursor is not directly over an icon. Check the grammar / spelling of the text.
 * Repeat with the Conservation Practices. We don't have the summaries yet (see issue https://github.com/WikiWatershed/model-my-watershed/issues/494), but for testing purposes 'Green Roof' has a summary which just says 'Green Roof Summary...'. This should be removed before merging. 
 * Check that clicking a modification allows you to draw a polygon.

Connects #435
